bindings/matrix-sdk-crypto-js/src/device.rs

```diff
@@ -196,6 +196,17 @@ impl Device {
     pub fn first_time_seen(&self) -> u64 {
         self.inner.first_time_seen_ts().0.into()
     }
+
+    /// Mark this device as verified.
+    ///
+    /// Returns a signature upload request that needs to be sent out.
+    pub fn verify(&self) -> Promise {
```

Comment:
Perhaps some more docs are warranted here, i.e. this method doesn't work for devices that aren't our own.

The docs on the main crate do explain this: https://matrix-org.github.io/matrix-rust-sdk/matrix_sdk/encryption/identities/struct.Device.html#method.verify.

Reply:
Added